sub-pages/_single.html

```diff
@@ -95,4 +98,7 @@ <h2 class="h3">
     </aside>
     {% endif %}
 
+    {% set office = vars.office %}
```

Comment:
Yeah having it in the includes file where it's used seems like it would be clearer.